app/routes/questions/search.tsx

```diff
@@ -15,6 +15,10 @@ export function search(question: string, onlyLive: boolean) {
     onlyLive ? 1 : 0
   }`
   return fetch(url).then(async (response) => {
+    if (response.status != 200) {
+      console.error(await response.text())
+      return new Response('Error while searching for questions', {status: 502})
```

Comment:
I considered returning what the downstream server returns, but that's usually random HTML, so decided against it

Reply:
please do - we don't have to show it in UI, but at least make it visible in network tab of the browser

`console.error` from here is going to CloudFlare, not very useful for everyday debugging 😅

Reply:
(and no need for nice formatting of error in UI)